### PR TITLE
Add regex matching for pastebin urls to send the raw version

### PR DESCRIPTION
**Pull request by jab416171**

### CODE REVIEW

**Comment by Dorukyum** [comment]
Hello, I think it would be better if we did the following:
1. Check if the guild is Pycord (`if message.guild == self.bot.pycord`)
2. List the links and send in one message (to prevent the bot from spamming)

File: cogs/pycord.py [hunk 1]
```diff
@@ -104,6 +107,18 @@ async def update_staff_list(self, ctx: Context):
...
+        if message.guild == self.bot.pycord:
+            messages = []
+            matches = re.findall(PASTEBIN_RE, message.content)
+            for match in matches:
+                base_url = match[0]
+                paste_id = match[1]
+                messages.append(f"{base_url}/raw/{paste_id}")
+            if messages:
+                await message.channel.send("\n".join(messages))
+
```
Comment (ephemeral8997) [inline cogs/pycord.py:120]: You should remove the extra empty line, but the code looks pretty good.

**Review by ephemeral8997 — APPROVED** [review]

**Review by IlluminatiFish — CHANGES_REQUESTED** [review]
Could maybe use a more specific regex to match pastebin urls to prevent FPs

Suggested regex:
`(https?://pastebin.com)/([a-zA-Z0-9]{8})`

Also you could tuple unpack the `match` variable as it will always be a pair of values (`base_url` & `paste_id`)

File: cogs/pycord.py [hunk 2]
```diff
@@ -5,6 +5,9 @@
 
 from utils import Cog, pycord_only
 
+import re
+
+PASTEBIN_RE = re.compile(r"(http(?:s?)://pastebin.com)(?:/)(\w+?)(?!/\w+)\W")
```
Comment (Dorukyum) [inline cogs/pycord.py]: Suggestion by @IlluminatiFish 
```suggestion
PASTEBIN_RE = re.compile(r"(https?://pastebin.com)/([a-zA-Z0-9]{8})")
```

**Review by IlluminatiFish — APPROVED** [review]
lgtm

**Comment by Dorukyum** [comment]
I tested this and it works well but I'm thinking, why/how would this be useful?

**Comment by jab416171** [comment]
pastebin's default landing page has ads and is slow and obnoxious. the raw paste is just the text.
Chicc said ["we only accept pastebin links"](https://canary.discord.com/channels/881207955029110855/881309496385884180/952289683600183426) and it inspired me to add this feature.

**Review by Dorukyum — APPROVED** [review]
Looks good, thanks for your contribution

**Pull request merged**